pkg/network/telemetry/ebpf_telemetry.go

```diff
@@ -224,6 +244,12 @@ func (b *EBPFTelemetry) initializeHelperErrTelemetryMap(probes []*manager.Probe)
 	h := fnv.New64a()
 
 	for _, p := range probes {
+		// Some hook points, like tcp_sendmsg, are probed in
+		// multiple different programs.
+		if _, ok := b.probeKeys[p.EBPFFuncName]; ok {
```

Comment:
I realize it isn't a change in this PR, but we do have the `UID` field to disambiguate multiple probes on the same function.

Reply:
That is a good point. I will make a followup PR for this.